packages/core/src/ccss.ts

```diff
@@ -27,7 +27,7 @@ const generate = (v: Partial<CCSSProps>, options): string => {
     return generated
 }
 
-export const defaultOptions = createOptions()
+export const defaultOptions = createOptions<any>()
```

Comment:
No any please. Use <T extends CCSSProps>

Reply:
Actually, there should not be type here,  it should see correctly cretaOptions's types. You might need to add an overlad type to handle the case when no args incoming.

Reply:
I ended up having this type on the createOption function declaration. Then I removed the type from this line.